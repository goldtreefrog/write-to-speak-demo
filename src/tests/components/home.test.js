import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import Home from "./../../components/home.js";
import { createStore } from "redux";

it("renders without crashing and matches snapshot", () => {
  // Create store for shallow testing below. (Compare with approach used in Talk, which differs.)
  const initialState = { auth: { currentUser: null } };
  const store = createStore(function(state = initialState, action) {
    return state;
  });

  const wrapper = shallow(<Home store={store} />);
  expect(wrapper).toMatchSnapshot();
});
