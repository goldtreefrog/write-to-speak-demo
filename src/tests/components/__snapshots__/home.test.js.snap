// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing and matches snapshot 1`] = `
<h1>
  Home Page
</h1>
`;
